readthedocs/core/context_processors.py

```diff
@@ -10,7 +10,7 @@ def readthedocs_processor(request):
     exports = {
         'PUBLIC_DOMAIN': settings.PUBLIC_DOMAIN,
         'PRODUCTION_DOMAIN': settings.PRODUCTION_DOMAIN,
-        'USE_SUBDOMAINS': settings.USE_SUBDOMAINS,
+        'USE_SUBDOMAINS': settings.USE_SUBDOMAIN,
```

Comment:
We should rename the key too (I just checked, and there is not other places using this)

Reply:
sure!

Reply:
@stsewd Done.:)